src/zino/tasks/bgpstatemonitortask.py

```diff
@@ -290,21 +291,18 @@ def _update_single_bgp_entry(self, oid, row: dict[str, Any], local_as: str, upti
         if data.peer_remote_address == "32.1.7.0":
             return
 
+        # Internal bgp sessions are not observed
+        if local_as == data.peer_remote_as:
+            return
+
         index = f"{self.device.name},{oid}"
 
         if data.peer_state == "established":
-            handled = False
             bgp_peer_up_time = self.device_state.bgp_peer_up_times.get(data.peer_remote_address, None)
-            if (
-                bgp_peer_up_time
-                and uptime >= bgp_peer_up_time
-                and bgp_peer_up_time > data.peer_fsm_established_time
-                and local_as is not data.peer_remote_as
-            ):
+            if bgp_peer_up_time and uptime >= bgp_peer_up_time and bgp_peer_up_time > data.peer_fsm_established_time:
```

Comment:
```suggestion
            if bgp_peer_up_time and uptime >= bgp_peer_up_time > data.peer_fsm_established_time:
```
I think its valid to simplify like this. I'll let you decide if its a good choice, whatever gives best readability should be used